sources/packages/backend/apps/api/src/services/student/student.service.ts

```diff
@@ -228,6 +230,14 @@ export class StudentService extends RecordDataModelService<Student> {
         await entityManager
           .getRepository(SFASRestriction)
           .update({ individualId: sfasIndividual.id }, { processed: false });
+
+        // If there is a match in SFAS, record that information is available from legacy system with a note attached to the student.
+        this.addStudentNote(
+          student.id,
+          NoteType.System,
```

Comment:
Please, confirm if this is a "System"  or "General" note type. The ticket says: "Add 'general' note to student profile". It's not clear to me.

Reply:
Probably is meant to be 'general'! I'd focused on the 'system-generated note' comment. Will update.

Reply:
FMU: What is the difference b/w `System` and `General` notes?

Reply:
@sh16011993 `System` and `General` are just categories under the notes.
![image](https://github.com/bcgov/SIMS/assets/61259237/bcf76440-a095-405a-8cc8-7dc78f59ae0c)

The request from the business was to add it under the `General` category, this was the AC and we should follow it unless another decision is taken. 
We are using the system user because it seemed to us not consistent to use the student user while saving this particular note. This was the decision during the technical ACs creation.

Reply:
Thanks @andrewsignori-aot